static/js/common.js

```diff
@@ -58,8 +58,25 @@ function password_quality(password, bar, password_field) {
     return acceptable;
 }
 
+function password_warning(password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
```

Comment:
@adnrs96
`
min_password_length = password_field.data('minLength') || min_password_length;
`
The value of password_field.data('minLength') is populated from settings.PASSWORD_MIN_LENGTH. So var min_password_length = 6 is taken if the value is not specified in settings. It's in the same format as the `password_quality` function in the same file.